README.md

```diff
@@ -21,8 +21,8 @@ The available Go binaries
 |  `BUILDKITE_PARALLEL_JOB` | - | Required, test plan for specific node |
 | `BUILDKITE_SPLITTER_SUITE_TOKEN` | `BUILDKITE_ANALYTICS_TOKEN` | Required, unique token for Test Suite that is being parallelised |
 |  `BUILDKITE_SPLITTER_IDENTIFIER` | `BUILDKITE_BUILD_ID` | Required, use default unless you have a specific use case |
-| `BUILDKITE_SPLITTER_TEST_FILE_PATTERN` | `spec/**/*_spec.rb` | Optional, glob pattern for discovering test files that need to be executed. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library*. |
-| `BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN` | - | Optional, glob pattern to use for excluding test files or directory. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library.* |
+| `BUILDKITE_SPLITTER_TEST_FILE_PATTERN` | `spec/**/*_spec.rb` | Optional, file glob pattern for discovering test files that need to be executed. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library*. |
+| `BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN` | - | Optional, file glob pattern to use for excluding test files or directory. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library.* |
```

Comment:
It says `for excluding test files or directory.` -- so can I use the pattern to exclude a whole directory tree? You suggested in #52 that it had to match files.

Reply:
Hi @TJC, yes you can use `BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN` to exclude a whole directory, providing you use the correct globing pattern syntax. The issue you were having in #52 seems to be caused by the issue in the identifier that you raised in #55.

Can you confirm that the exclude pattern is now working on your side?

Reply:
I was referring to the comment in https://github.com/buildkite/test-splitter/issues/52#issuecomment-2091992853 which said "We expect the exclude pattern to take the form of a file glob, not a directory path as you're probably used to doing with Rspec!"

The description you're changing in the PR now says "file glob pattern to use for excluding test files or directory." 
That seems to imply you could pass a glob that matched a directory.

So perhaps you can see where this documentation isn't actually that clear about whether it accepts a directory, or if you actually need to be creating a glob that matches all files within a directory. Based on our discussions I think the answer is "a glob that matches all files within a directory", but to a fresh reader who comes to this project, I don't think that would be clear.

Reply:
There's also been #58 merged since this PR was opened, which I think has changed how it works?